libs/icelandic-names-registry/types/src/lib/types.ts

```diff
@@ -0,0 +1,28 @@
+export type IcelandicName = {
+  id: number
+  icelandic_name: string
+  type: string
+  status: string
+  visible: boolean
+  description: string
+  url: string
+  created: Date
+  modified: Date
+}
+
+export enum EnumNameType {
+  ST = 'ST',
+  DR = 'DR',
+  MI = 'MI',
+  RST = 'RST',
+  RDR = 'RDR',
+}
+
+export enum EnumStatusType {
+  ST = 'Haf',
+  DR = 'Sam',
+  OAF = 'Óaf',
+}
+
+export type NameType = 'ST' | 'DR' | 'MI' | 'RST' | 'RDR'
+export type StatusType = 'Haf' | 'Sam' | 'Óaf'
```

Comment:
These types seem to be defined in the icelandic names registry app here`apps/icelandic-names-registry/backend/types/index.ts`.
Consider removing either that file or this file to prevent bot from being used